backend/capellacollab/projects/toolmodels/restrictions/models.py

```diff
@@ -0,0 +1,38 @@
+# SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+import typing as t
+
+from pydantic import BaseModel
+from sqlalchemy import Boolean, Column, ForeignKey, Integer
+from sqlalchemy.orm import relationship
+
+from capellacollab.core.database import Base
+
+if t.TYPE_CHECKING:
+    from capellacollab.projects.toolmodels.models import DatabaseCapellaModel
+
+
+class ToolModelRestrictions(BaseModel):
+    # If true, access to the specific resource is granted for a model.
+    # If false, the access is not allowed.
+
+    allow_pure_variants: bool
+
+    class Config:
+        orm_mode = True
```

Comment:
Still I'm wondering if this justified a separate table. We could also have a field `capabilities`/`features` or something like that, as an array of extra stuff to enable for the model.

Reply:
I described it in the PR description. Have you seen it? 
The option with postgres arrays is something that I didn't consider, but it's a better solution indeed.